src/dice/ffi/metall.cpp

```diff
@@ -1,120 +1,63 @@
 #include <dice/ffi/metall.h>
 #include <dice/ffi/metall_internal.hpp>
 
-namespace dice::metall_ffi::internal {
-    dice::sparse_map::sparse_map<std::string, std::unique_ptr<metall_manager>> metall_managers;
```

Comment:
The background why we did it that way in the first place was to prevent opening the same metall metall storage twice. But since that should never happen in our use-cases we also can get rid of the additional map. Please also remove the dependency in the conanfile.py